### ISSUE TITLE
[TTPReq] Education: How to handle PIN entry for a card

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14166]
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

##  How to handle PIN entry for a card

- Use the visual assets from [Marketing Guidelines](https://developer.apple.com/tap-to-pay/marketing-guidelines/)
- Use the latest bezels
- Use localized images for all the supported TTP markets
- Only integrate within required markets 

## Designs

|  |  |
|---------|---------|
| ![Image 1](https://github.com/user-attachments/assets/f352b997-d4bc-41dd-9e22-de584f4750ff) | ![Image](https://github.com/user-attachments/assets/68480a7e-e50b-46c3-bcbc-cffe7e8d168e)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by staskus** [comment]
Moved out of scope since it's only required for Canada and Interac
